src/System.IO.FileSystem/tests/File/Copy.cs

```diff
@@ -48,6 +49,20 @@ public void CopyOntoSelf()
             Assert.Throws<IOException>(() => Copy(testFile, testFile));
         }
 
+        [DllImport("libc", SetLastError = true)]
+        private static extern int symlink(string target, string linkpath);
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        public void DanglingSymlinkCopy()
+        {
+            var dangling_symlink = GetTestFileName();
+            var missing_target = GetTestFileName();
```

Comment:
Style: `var` can only be used with `new`, cast, and `as`-cast expressions. (The type name has to be on the line, and not as a generic parameter or part of a member name)